src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Controllers/AuthenticationController.cs

```diff
@@ -485,6 +495,18 @@ private static string GetOrganisationNumberFromConsumerClaim(ClaimsPrincipal ori
             return organisationNumber;
         }
 
+        private static bool HasServiceOwnerScope(ClaimsPrincipal originalPrincipal)
```

Comment:
what happens if they have a multipurpose client that holds multiple scopes? Is that even possible? e.g. altinn scopes and skd scopes for the same client?

Reply:
I believe that would be a normal scenario. But for Altinn use they need to define two different clients. One for org usage and for end user usage @elsand  do you have any thoughs on this?

Reply:
My main consern is if they have multiple scopes on the client, and the first one isn't an altinn:serviceOwner claim, although it is present in the list. Would the client only have the altinn scopes, or could the serviceOwner client also have other scopes?

Reply:
nvm, seems all the scopes will be present in the string that is searched.